src/frontend/planner_test/tests/testdata/insert.yaml

```diff
@@ -234,3 +234,86 @@
     create table t (a int, b int);
     insert into t values (0,1), (1,2) returning sum(a);
   binder_error: 'Bind error: should not have agg/window in the `RETURNING` list'
+- name: insert and specify all columns with values
+  sql: |
+    create table t (a int, b int);
+    insert into t (b, a) values (1, 2);
+  logical_plan: |
+    LogicalInsert { table: t }
```

Comment:
Do you think it's worthwhile to also show the column mapping (or default values) here? So that we can know what's actually happening in the `Insert`.

We can show this only in `verbose` mode.

Reply:
good idea, will work on it